tests/e2e.rs

```diff
@@ -1,17 +1,17 @@
 #![cfg(test)]
```

Comment:
We should add one more test, to assert that SSO linking actually works. I *believe* Zitadel is set up to allow this already, we just need to add a test.